data/global/Excel/skills.txt

```diff
@@ -376,4 +376,4 @@ proc_SplashDamage_Three	373		proc_SplashDamage_Three	4	8													proc_splash
 proc_SplashDamage_Four	374		proc_SplashDamage_Four	4	8													proc_splashdamage_four																																																																																										11	17						proc_splashdamage_four												1	7		rng		miss	miss									A1	A1	xx				1																1										1															1						1	8	4	1	1	1										min(24,ln12)	# missiles	par3	activation frame	2	triggered							11	number of arrows to fire	1	additional arrows per level	1	Acivation frame of arrows.																			1								8	128																																			384	3000	0
 Mark of the Bear	375		mark of the bear		18																			markbear		4500		attackrate	par1	damageresist	par2																																																																																																	0		none																		1																1				1						1									2	bear					0	0																																25	Attack Speed %	20	Damage Resist %																					1								8																																				256	1000	0
 Mark of the Wolf	376		mark of the wolf		18																			markwolf		4500		item_tohit_percent	par1	item_maxhp_percent	par2																																																																																																	0		none																		1																1				1						1									2	wolf					0	0																																30	Attack Rating %	40	Life %																					1								8																																				256	1000	0
-proc_SplashDamage_Four	377		proc_SplashDamage_Five	4	8													proc_splashdamage_five																																																																																										11	17						proc_splashdamage_five												1	7		rng		miss	miss									A1	A1	xx				1																1										1															1						1	8	4	1	1	1										min(24,ln12)	# missiles	par3	activation frame	2	triggered							11	number of arrows to fire	1	additional arrows per level	1	Acivation frame of arrows.																			1								8	128																																			384	3000	0

```

Comment:
Looks like this may have reverted?

Reply:
Looks ok on my end? Four changed to Five.
There is a separate pull req for just this change but figured I'd merge it here if I was doing a separate update that impacted skills.txt